backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,85 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-no-phi-views
+      author: josh@skylight.digital
+      comment: Add views for some tables that omit columns containing PHI.
```

Comment:
Do we want a view for the audit log table? I feel like some folks are doing analytics queries based on it at this point, but it definitely has PHI exposure right now.

Reply:
@TimBest thoughts? These views are just following the criteria in #906 at the moment.

Reply:
We will need a view on the audit log but it shouldn't block getting a read only user and a best effort hide PII. It is okay if we lose some analytical capabilities in the short term. 

For context I believe this ticket was created before the audit log was a thing.

Reply:
Sounds good - tabling this for a future ticket then.